docs/creators.md

```diff
@@ -331,7 +331,7 @@ Labware creators are responsible for creating new labware from a parent labware.
 {include:LabwareCreators::PooledTubesFromWholeTubes}
 
   Used directly in 2 purposes:
-  LBSN-9216 Lib PCR Pool and GBS MiSeq Pool
+  GBS MiSeq Pool and LBSN-9216 Lib PCR Pool
```

Comment:
Odd how it's rearranged the orders in a lot of these doc files..

Reply:
As it builds these docs automatically I assumed it had some logic to it, and it will likely undo all your changes the next time it runs for another developer. We're not going to sort these files manually each time we change something. Wonder if it's pulling them out in the order they appear in the codebase files?